src/main/java/edu/gonzaga/EpidemicCard.java

```diff
@@ -2,7 +2,56 @@
 
 public class EpidemicCard extends Card {
     
-    public void triggerEpidemic(){
+    /**
+     * Constructor for the EpidemicCard class
+     * @return an EpidemicCard object
+     * 
+     * @Author Tony
+     */
+    public void triggerEmpidemic(Gameboard board) {
+        // increase the infection rate
+        increaseInfectionRate(board);
 
+        // infect
+        infect(board);
+
+        // intensify
+        intensify(board);
+    }
+
+    /**
+     * Infects cities
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void infect(Gameboard board) {
+        // infect cities
+        board.takeInfectionTurn();
+    }
+
+    /**
+     * Intensifies the game
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void intensify(Gameboard board) {
+        // shuffle the discard pile and add it to the draw pile
+        board.getPlayerDeck().putShuffledDiscardPileOnTop();
```

Comment:
intensify() should not take the player deck, it should get the **infection deck** discard pile and put it back on top of the **infection deck**.